content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -407,22 +396,32 @@ Then create the check using sensuctl and the `disk_usage-check.json` file includ
 sensuctl create --file disk_usage-check.json
 {{< /highlight >}}
 
-We should see it working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
+_**NOTE**: You don't need to make any changes to disk_usage-check.json before running `sensuctl create --file disk_usage-check.json`._
+
+You should see the check working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
 
 {{< highlight shell >}}
 sensuctl event list
 {{< /highlight >}}
 
-Now we should be able to see disk usage metrics for the sandbox in [Grafana](http://localhost:4002/d/go02/sensu-go-sandbox-combined).
+Now, you should be able to see disk usage metrics for the sandbox in Grafana: refresh your Grafana tab or [reload with this link](http://localhost:4002/d/go02/sensu-go-sandbox-combined).
+
+You made it! You're ready for the next level of Sensu-ing. Here are some resources to help continue your journey:
+
+- Try another [lesson in the Sensu sandbox][7]
+- [Install Sensu Go][8]
+- [Collect StatsD metrics][9]
+- [Create a ready-only user][10]
```

Comment:
Maybe we should end this lesson with a command that deletes all the resources we created (a la https://docs.sensu.io/sensu-go/5.13/sensuctl/reference/#deleting-resources) -- or will subsequent lessons rely on these resources in some way?

Reply:
I don't believe our lessons, at least in the sandbox, depend on previous resources. So that would be a good add I think, but it would probably require some additions to the other guides like "If you did this lesson, you can skip these things...". Probably out of scope for this PR.

Reply:
Something I can look into in a follow-up...

Reply:
I think we can resolve this section with having a `vagrant destroy` as suggested for:

https://github.com/sensu/sensu-docs/pull/1833#discussion_r336226957

We actually don't clean up the VM at all, so new users of Vagrant might be surprised to find that VM running later on.